packages/fast-layouts-react/src/pane/pane.tsx

```diff
@@ -130,7 +130,7 @@ class Pane extends Foundation<PaneProps, IPaneState> {
             width: 300
         };
 
-        this.onMouseMove    = throttle(this.onMouseMove, 16);
+        this.onMouseMove    = throttle(this.onMouseMove, 1);
```

Comment:
Does this improve the speed a significant amount? This means it will be firing every millisecond.